packages/core/core/src/Strapi.ts

```diff
@@ -306,14 +306,14 @@ class Strapi extends Container implements Core.Strapi {
     return this.get('hooks').get(name);
   }
 
-  // api(name) {
-  //   return this.get('apis').get(name);
-  // }
-
-  get api(): Record<string, Core.Module> {
+  get apis() {
```

Comment:
Not typed anymore? Does it infer the type from getAll?

Reply:
it's typed in the core types :)

Reply:
Not super super consistent with the other method then :confused: